xml/System.Reflection.Metadata.Ecma335/MethodBodyStreamEncoder.xml

```diff
@@ -106,13 +106,13 @@
         <Parameter Name="hasDynamicStackAllocation" Type="System.Boolean" Index="4" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="instructionEncoder">To be added.</param>
-        <param name="maxStack">To be added.</param>
-        <param name="localVariablesSignature">To be added.</param>
-        <param name="attributes">To be added.</param>
-        <param name="hasDynamicStackAllocation">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="instructionEncoder">The instruction encoder.</param>
+        <param name="maxStack">The max stack.</param>
+        <param name="localVariablesSignature">The local variables signature handle.</param>
+        <param name="attributes">The method body attributes.</param>
+        <param name="hasDynamicStackAllocation"><see langword="true" /> if the method allocates from dynamic local memory pool (the IL contains localloc instruction); <see langword="false" /> otherwise.</param>
```

Comment:
```suggestion
        <param name="hasDynamicStackAllocation"><see langword="true" /> if the method allocates from the dynamic local memory pool (the IL contains the <see langword="localloc" /> instruction); <see langword="false" /> otherwise.</param>
```